### ISSUE TITLE
@needs does not seem to work on Windows 10

### ISSUE THREAD

**Issue by jxanders** [paver/paver#209]
This is really weird. My peer has no issue on his Mac, but for some reason the needs decorator doesn't seem to work on Windows 10. I'm not getting any errors other than the task is simply not running.

This is how the pavement.py is set up
```
@task
def build():

@task
@needs('build')
def package():
```
When I run `paver package`, I get an error about a folder not existing. This folder is created by the build task. None of the prints fire for the build task either.

Peer's correct output:
```
---> pavement.package
---> pavement.build
```

My output:
```
---> pavement.package
attention-pc-v0.1.0.zip

Captured Task Output:
---------------------
---> pavement.package
(missing file error stripped for shortness)
```

**Comment by jxanders** [comment]
PEBCAK

**Comment by Almad** [comment]
@jxanders Would you elaborate on what the problem was? For one, for other people having the issue, but also if there's something to update in either doc or error messages to make it more clear.

**Comment by jxanders** [comment]
Too many hours sitting at the computer. My peer pushed changes to dependencies etc and I was reviewing the PR and cloned a fresh copy of the code. I tested installing the dependencies, etc. He forgot to put the @needs tag, but realized it and fixed it and so when I checked on Gitlab the tag was there. Just poor timing on my part in terms of pulling the code seconds before he pushed that fix. The fix did end up in the other repo which is why I thought I was seeing it there, but then I realized my mistake...
